cmd/codecovgen/README.md

```diff
@@ -0,0 +1,3 @@
+# codecovgen
+
```

Comment:
Out of ignorance: Do we want to include the autogenerated status header in the readme?

Reply:
I tried adding this, but it does not seem like mdatagen generates this.